packages/replay/test/integration/session.test.ts

```diff
@@ -329,18 +207,35 @@ describe('Integration | session', () => {
     });
 
     // `_context` should be reset when a new session is created
-    expect(replay.getContext()).toEqual(
-      expect.objectContaining({
+    expect(replay.getContext()).toEqual({
+        earliestEvent: null,
         initialUrl: 'http://dummy/',
         initialTimestamp: newTimestamp,
-      }),
-    );
+        urls: [],
+        errorIds: new Set(),
+        traceIds: new Set(),
+    });
   });
 
-  it('does not record if user has been idle for more than MAX_SESSION_LIFE and only starts a new session after a user action', async () => {
+  it('should have a session after setup', () => {
+    expect(replay.session).toMatchObject({
+      lastActivity: BASE_TIMESTAMP,
+      started: BASE_TIMESTAMP,
+    });
+    expect(replay.session?.id).toBeDefined();
+    expect(replay.session?.segmentId).toBeDefined();
+  });
+
+  it('clears session', () => {
+    clearSession(replay);
+    expect(WINDOW.sessionStorage.getItem(REPLAY_SESSION_KEY)).toBe(null);
+    expect(replay.session).toBe(undefined);
+  });
+
+  it('creates a new session if current session exceeds MAX_SESSION_LIFE', async () => {
```

Comment:
New test to ensure we start a new session if we are active and exceed MAX_SESSION_LIFE